lib/connectors/SentinelConnector/index.ts

```diff
@@ -25,6 +25,7 @@ type PreferredSlaves =
 interface ISentinelConnectionOptions extends ITcpConnectionOptions {
   role: 'master' | 'slave'
   name: 'string'
+  sentinelPasseword?: 'string'
```

Comment:
Typo here. `sentinelPasseword` should be `sentinelPassword` I guess?

Reply:
I'm sorry. It is typo. fixed.